src/content/smart-bulletins/030.md

```diff
@@ -0,0 +1,88 @@
+- Ratio of reviewing officials to fleet cards (span of control); and
+- Number of fleet cards with transaction limits over micro-purchase threshold. Integrated cards only (to be provided to CCCM by CFO Act Agencies):
+- Ratio of A/Os to integrated cardholders (purchase business line - span of control);
+- Ratio of reviewing officials to integrated cards (fleet business line - span of control);
+- Number of integrated cards with transaction limits over the micro-purchase threshold and
+- Ratio of the number of confirmed violations reported pursuant to P.L. 112-194 as compared to the number of valid transactions within the reporting period (GSA's Center for Charge Card Management will obtain confirmed violation information from OMB/OFFM).
+
+### OMB Circular A-123, Appendix B Chapter 5.3.2 – Narrative Reporting – 
+For agencies/organizations subject to the CFO Act, the following information should be updated annually on a fiscal year basis and provided to CCCM *no later than 90 calendar days after the end of each fiscal year:*
+- The date(s) of most recent and next scheduled independent review(s) (e.g., OIG audits and risk assessments) for all agency charge card programs;
+- A description of the current process for monitoring delinquency, including what reports the agency reviews and what actions are taken when a problem is discovered;
+- A description of any best practices the agency employs in charge card management;
+- Progress reports on agency integration of GSA SmartPay e-payable payment process into their accounts payable processes; and
+- Any additional information regarding agency management and use of charge card programs, including innovative applications to streamline business processes, increased revenue generation from expanded use for payments, and creative internal controls that maintain program integrity, flexibility and ease of use.
+
+The due date for submission of FY2019 agency narrative reporting to CCCM is similarly
+extended to NLT February 28, 2020. This extension applies only to FY 19 reports. For
+FY2020 reports and beyond, agency narrative reports are due to CCCM NLT 90
+calendar days after the end of the fiscal year requirement as specified in the current
+version of OMB Circular A-123, Appendix B.
+
+
+## Action
+
+Agencies shall utilize the reporting templates attached below to submit the required agency/organization statistical and narrative reporting data. Agency reports should be emailed to OMBA123AppBReporting@gsa.gov using the following file naming convention: <span style="color:#4f81bd">AgencyName_OMB_A123_FiscalYear</span> (e.g., GeneralServicesAdministration_OMB_A123_2019). Further instructions for report submission are provided in the templates. In the event any sensitive data is to be provided in these reports, agencies/organizations are reminded to encrypt the files and take any other security precautions as required by agency/organization policy.
```

Comment:
Agencies shall utilize the reporting templates attached below to submit the required agency/organization statistical and narrative reporting data. Agency reports should be emailed to OMBA123AppBReporting@gsa.gov using the following file naming convention: AgencyName_OMB_A123_FiscalYear (e.g., GeneralServicesAdministration_OMB_A123_2019). Further instructions for report submission are provided in the templates. In the event any sensitive data is to be provided in these reports, agencies/organizations are reminded to encrypt the files and take any other security precautions as required by agency/organization policy.